### PR TITLE
minor update for plugin tutorial

### PR DESCRIPTION
**Pull request by ltalirz**

### CODE REVIEW

File: docs/sections/plugins/index.rst [hunk 1]
```diff
@@ -7,7 +7,7 @@ Plugins
 
     If you haven't done so already, please **first** watch the `introduction to the AiiDA plugin system <https://www.youtube.com/watch?v=bjTUnHXZ6oY&list=PL19kfLn4sO_-e_A9lVYb_NBNcwoVvUP6V>`_ (~20 minutes).
 
-    For this session, we suggest two options depending on your interests and experience -- please post in the chat which of the two (A/B) you are going to follow:
+    For this session, pick one of two options depending on your interests and experience:
 
 .. panels::
     :header: header-text
```
Comment (mbercx) [inline docs/sections/plugins/index.rst]:
```suggestion
    :header: panel-header-text
```

File: docs/sections/plugins/index.rst [hunk 2]
```diff
@@ -32,7 +32,8 @@ Plugins
         :header-rows: 0
 
         * - |time| 1h30m
-          - Intermediate
+          - |aiida| :aiida-blue:`Intermediate`
+
 
 .. panels::
     :header: header-text
```
Comment (mbercx) [inline docs/sections/plugins/index.rst]:
```suggestion
    :header: panel-header-text
```

**Review by mbercx — DISMISSED** [review]
All good, thanks Leo! Just two minor comments related to the fact that I changed the `header-text` CSS a bit, so now there is a specific `panel-header-text` class that doesn't have this extra `margin-bottom`.

**Review by ramirezfranciscof — APPROVED** [review]
LGTM! Thanks @ltalirz and @mbercx !

**Pull request merged**